rust/core-lib/src/file.rs

```diff
@@ -254,38 +254,46 @@ fn try_load_file<P>(file_manager: &FileManager, buffer_id: BufferId, path: P) ->
     let mut new_tail_details = TailDetails::default();
     let mut bytes = Vec::new();
 
-    let file_info = file_manager.get_info(buffer_id);
-    match file_info {
-        Some(v) => {
-            let is_tail_enabled = v.tail_details.is_tail_enabled;
-            if is_tail_enabled {
-                debug!("Tailing file");
-                let end_position =
-                    f.seek(SeekFrom::End(0)).map_err(|e| FileError::Io(e, path.as_ref().to_owned()))?;
-                let current_position = v.tail_details.current_position_in_tail;
-
-                let diff = end_position - current_position;
-                bytes = vec![0; diff as usize];
-                f.seek(SeekFrom::Current(-(bytes.len() as i64))).unwrap();
-                f.read_exact(&mut bytes).unwrap();
-
-                new_tail_details = TailDetails {
-                    current_position_in_tail: end_position,
-                    is_tail_enabled: true,
-                    is_at_bottom_of_file: true,
-                };
-            } else {
-                debug!("Tail is false, So loading entire file.");
+    #[cfg(feature = "notify")]
```

Comment:
`try_load_file` is common for fuchsia and non-fuchsia. Is this correct way to handle this?

Reply:
Well how I'd think about this is, what in the following block works or doesn't work without the `notify` feature? From just skimming the code, it looks like there's no use of `notify` at all; We're just referencing some types that are themselves hiding behind `cfg(feature = "notify")`. Is this necessary? For instance, we could _always_ have `tail_details` exist, even if notify was disabled; it just wouldn't do anything, and I think that's probably fine?